nginx/start.sh

```diff
@@ -2,8 +2,16 @@
 set -e
 
 export IMMICH_WEB_URL=${IMMICH_WEB_URL:-http://immich-web:3000}
+IMMICH_WEB_SCHEME=$(echo "$IMMICH_WEB_URL" | grep -Eo '^https?://' || echo "http://")
+export IMMICH_WEB_SCHEME
+IMMICH_WEB_HOST=$(echo "$IMMICH_WEB_URL" | cut -d '/' -f 3)
```

Comment:
Can you not export on this same line?

Reply:
https://github.com/koalaman/shellcheck/wiki/SC2155